client/components/Login.jsx

```diff
@@ -0,0 +1,73 @@
+import { useState } from 'react';
+import axios from 'axios';
+import styles from './Login.module.css';
+
+const Login = () => {
+
+    const [inputEmail, setInputEmail] = useState('');
+    const [inputPassword, setInputPassword] = useState('');
+    const [validEmail, setValidEmail] = useState(false);
+    const [validPassword, setValidPassword] = useState(false);
+
+
+    const onSubmitHandler = (event) => {
+        event.preventDefault();
+        postResponse();
+    };
+
+    const postResponse = async () => {
+        const formData = { email: inputEmail, password: inputPassword }
+        try {
+            const res = await axios.post('/api/login', formData);
+            console.log(res);
+        } catch (err) { console.log(err); }
+    };
+
+    const validateEmail = (email) => {
```

Comment:
I see a repeating pattern between `validateEmail` and `validatePassword` .
Do you think we can turn them into one function and reuse it for email and password ? WDYT ?